### PR TITLE
Events feature

### PR DESCRIPTION
**Pull request by Mohitkumar33**
Made the countDown feature,
User can set an event for any meeting or any birthday party. 
Countdown will be shown from the date of creation of the event.
Data doesn't go on refresh.
countdown feature

### CODE REVIEW

File: src/components/small components/addEvents.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.enter-event-box {
+  height: 12.5rem;
+  width: 13.5rem;
+  background-color: rgba(0, 0, 0, 0.637);
+  border-radius: 8px;
+  padding: 1rem;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  color: white;
```
Comment (deepak29-git) [inline src/components/small components/addEvents.css:10]: Use color variable

File: src/components/small components/addEvents.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  height: 2rem;
+  border: transparent;
+  border-radius: 15px;
+  margin: auto;
+  margin-top: 10px;
+}
+
+.close-create-event {
+  position: absolute;
+  height: 25px;
```
Comment (deepak29-git) [inline src/components/small components/addEvents.css:46]: Use rem unit

File: src/components/small components/displayEvents.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+.count-down span {
+  font-size: 0.5rem;
+}
+
+.single-event {
+  margin-bottom: 2rem;
+  position: relative;
+  padding-right: 1rem;
+  /* margin-right: 1rem; */
```
Comment (deepak29-git) [inline src/components/small components/displayEvents.css]: Remove commented code

**Review by deepak29-git — COMMENTED** [review]
Liked : folder structure is good and variable naming is also good

File: src/components/small components/addEvents.css [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+.enter-event-box input {
+  height: 1.8rem;
+  background-color: transparent;
+  border: transparent;
+  border-bottom: 1px solid white;
+  caret-color: white;
+  outline: none;
+  color: white;
```
Comment (nishasen) [inline src/components/small components/addEvents.css:25]: use variables instead of solid colors here

**Review by nishasen — APPROVED** [review]
Good use of localstorage, and good variable name, some of the places has repeated code, you can use map for that, also use variables for colors. Else your design looks good

**Pull request merged**